src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -1207,14 +1207,14 @@ static void window_cheats_paint(rct_window* w, rct_drawpixelinfo* dpi)
         }
         int32_t actual_month = _monthSpinnerValue - 1;
         gfx_draw_string_left(
-            dpi, STR_BOTTOM_TOOLBAR_CASH, gCommonFormatArgs, colour, w->windowPos.x + XPL(0) + TXTO,
-            w->windowPos.y + YPL(2) + TXTO);
+            dpi, STR_BOTTOM_TOOLBAR_CASH, gCommonFormatArgs, colour,
+            { w->windowPos.x + XPL(0) + TXTO, w->windowPos.y + YPL(2) + TXTO });
```

Comment:
When one of the objects is `ScreenCoordsXY` already, we're sticking to the following convention:
```suggestion
            w->windowPos + ScreenCoordsXY{ XPL(0) + TXTO, YPL(2) + TXTO });
```

This happens in other places in the PR as well, but I'll not open issues for all, please make sure to review the whole diff.